src/tests/lxc-test-unpriv

```diff
@@ -27,6 +27,42 @@ if [ $(id -u) -ne 0 ]; then
 	echo "ERROR: Must run as root."
 	exit 1
 fi
+
+# Test if we're using an overlayfs module that handles symlinks correctly. If
+# not, we skip these tests since overlay clones will not work correctly.
+modprobe -q overlayfs
+if [ "${?}" -eq 0 ]; then
```

Comment:
Why do this instead of just my suggested one line:

    if modprobe -q overlayfs; then